File: src/Package/Estadistica.java
package Package;

public class Estadistica {
    private int anio;
    private float recaudacion;

    public void setAnio(int anio) {
        this.anio = anio;
    }

    public void setRecaudacion(float recaudacion) {
        this.recaudacion = recaudacion;
    }

    public void mostrar() {
        System.out.println("------------------------------");
        System.out.println("Anio " + this.anio + " recaudacion " + this.recaudacion + "\n");
    }
}

File: Estadistica.cpp
#include "Estadistica.h"
#include <iostream>
using namespace std;

void Estadistica::setAnio(int anio)
{
	_anio = anio;
}

void Estadistica::setRecaudacion(float recaudacion)
{
	_recaudacion = recaudacion;
}
void Estadistica::mostrar() {
	cout << "------------------------------" << endl;
	cout << "Anio " << _anio << " recaudacion " << _recaudacion << endl;
}
